export/README.md

```diff
@@ -9,13 +9,23 @@ Code for exporting and printing files from the SecureDrop Qubes Workstation.
 
 ## Supported Printers
 
-TBD
+Printer support is currently limited to a subset of Brother and HP printers that offer USB
+(non-wireless) connectivity, such as:
+
+- HP LaserJet Pro 4001dn
+- Brother HL-L2320D
 
 ## Supported Export Devices
 
-We support luks-encrypted drives that are either MBR/DOS partitioned or GPT partitioned. If you use `Disks` in Linux to partition your drive, you can [follow these instructions](https://docs.securedrop.org/en/stable/set_up_transfer_and_export_device.html#create-usb-transfer-device) to create a new export device. You can also use [cryptsetup](https://linux.die.net/man/8/cryptsetup) to create a luks-encrypted device with full-disk encryption, for example:
+Export to LUKS-encrypted or VeraCrypt-encrypted USB devices is supported.
+
+### LUKS
+Provision an MBR/DOS partitioned or GPT partitioned LUKS-encrypted drive. If using
+`Gnome Disks` in Linux to partition your drive,
+[follow these instructions](https://docs.securedrop.org/en/stable/set_up_transfer_and_export_device.html#create-usb-transfer-device). LUKS-encrypted devices can also be created with [cryptsetup](https://linux.die.net/man/8/cryptsetup):
 
-1. `sudo cryptsetup luksFormat --hash=sha512 --key-size=512 DEVICE` where `DEVICE` is the name of your removable drive, which you can find via `lsblk -p`.
+1. `sudo cryptsetup luksFormat --hash=sha512 --key-size=512 DEVICE` where `DEVICE` is the name
+   of your removable drive, which you can find via `lsblk -p`.
```

Comment:
Nit: Can we standardize on either `lsblk -p` or `sudo fdisk -l` to keep the lookup consistent between methods?